components/avatar-component/README.md

```diff
@@ -0,0 +1,15 @@
+# Avatar
+
+Returns an image, default image if no src is supplied
+otherwise an image from supplied avatar data path and imgixHost
+storyteller name is for the title attribute
+
+## How to use
+
+`<avatar-component v-bind:avatar="storyteller" v-bind:avatar="storytellername" />`
```

Comment:
This does not correspond with the actual template

Reply:
changed to <avatar-component
  v-bind:avatar="page.storyteller.avatar"
  v-bind:name="page.storyteller.name"
  />

Reply:
<avatar-component
  v-bind:avatar="page.storyteller.avatar"
  v-bind:name="page.storyteller.name"
  />